src/Tab/Tab.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tab: Tab;
+};
+
+export const TabItem: FC<Props> = ({ tab }) => {
+  const { id, title } = tab;
+  const { tabId } = useParams();
```

Comment:
You've already used the `useParams()` inside your Tabs component. You can provide selected todo id via props